moped-editor/src/views/projects/projectView/ProjectActivityLogTableMaps.js

```diff
@@ -674,68 +619,10 @@ export const ProjectActivityLogCreateDescriptions = {
     label: (record, userList) =>
       userList[`${record.record_data.event.data.new.user_id}`] + " to the team",
   },
-  moped_proj_phases: {
-    label: (record, userList, phaseList) => {
-      const recordData = record.record_data.event.data.new;
-      const phaseName = phaseList[recordData?.phase_id] ?? "";
-      return `'${phaseName}' as Project Phase with start date as '${recordData.phase_start}' and end date as '${recordData.phase_end}'`;
-    },
-  },
   moped_project_files: {
     label: (record) =>
       `New file '${record.record_data.event.data.new.file_name}'`,
   },
-  moped_proj_milestones: {
-    label: (record, userList) => {
-      return (
-        fieldFormat(
-          record.record_data.event.data.new.milestone_description,
-          true
-        ) + " as a new milestone"
-      );
-    },
-  },
-  moped_proj_notes: {
-    label: (record, userList) => {
-      // remove HTML tags
-      const note = record.record_data.event.data.new.project_note.replace(
-        /(<([^>]+)>)/gi,
-        ""
-      );
-
-      const shortNote =
-        note.length > 30 ? note.substr(0, 30).trim() + "..." : note.trim();
-
-      return fieldFormat(shortNote, true) + " as a new note";
-    },
-  },
-  moped_proj_partners: {
-    label: (record, userList) => {
-      return (
-        fieldFormat(record.record_data.event.data.new.partner_name, true) +
-        " as a new partner"
-      );
-    },
-  },
-  moped_proj_components: {
-    label: (record, userList) => {
-      return (
-        fieldFormat(record.record_data.event.data.new.description, true) +
-        " as a new component"
-      );
-    },
-  },
-  moped_proj_funding: {
-    label: (record, userList) => {
-      //return '"' + record.record_data.event.data.new.funding_description + "\" as a new funding source";
-      return (
-        "A new funding source" +
-        (record.record_data.event.data.new.funding_description
-          ? ": " + record.record_data.event.data.new.funding_description
-          : "")
-      );
-    },
-  },
```

Comment:
thanks for cleaning up this file! ✨ i'll be more consistent about this in my future prs!!